### PR TITLE
BUG-100515 - more strict for ambari password

### PR DESCRIPTION
**Pull request by gregito**
making the cluster's ambari password more secure

### CODE REVIEW

File: core-api/build.gradle [hunk 1]
```diff
@@ -40,6 +40,7 @@ dependencies {
   testCompile group: 'org.springframework.boot',  name: 'spring-boot-starter',            version: springBootVersion
   testCompile group: 'org.springframework.boot',  name: 'spring-boot-starter-test',       version: springBootVersion
   testCompile group: 'junit',                     name: 'junit',                          version: junitVersion
+  testCompile group: 'org.glassfish.web',         name: 'javax.el',                       version: '2.2.6'
```
Comment (doktoric) [inline core-api/build.gradle]: do we using this one ?

Comment (gregito) [reply core-api/build.gradle]: yes, this dep provides the expression language implementation to be able to activate the annotation based validation of the related fields

Comment (lnardai) [reply core-api/build.gradle]: I think tomcat also has an implementation for javax.el.
https://mvnrepository.com/artifact/org.apache.tomcat/tomcat-el-api
Maybe worth to try it out, so we won't depend on glassfish.

Comment (gregito) [reply core-api/build.gradle]: I'll give it a shot, thank you!

**Pull request merged**